pcsx2/Patch.h

```diff
@@ -34,10 +34,9 @@
 // - The 0 cheats - cheats are enabled but nothing found/loaded from the "cheats" folder.
 // - The 6 widescreen patches are 6 pnach-style patch lines loaded either from cheats_ws folder or from cheats_ws.zip
 
-
```

Comment:
I think you mentioned this, but is it possible to avoid pulling in the console header by changing the files where Patch.h is included instead?

Reply:
It was pulling the console stuff from the gamedatabase, but that was a circular include :(